ci-operator/step-registry/hypershift/mce/kubevirt/create/disconnected/hypershift-mce-kubevirt-create-disconnected-commands.sh

```diff
@@ -0,0 +1,170 @@
+#!/bin/bash
```

Comment:
Let's start refactoring
- ci-operator/step-registry/hypershift/mce/kubevirt/create/hypershift-mce-kubevirt-create-chain.yaml
- ci-operator/step-registry/hypershift/kubevirt/create/hypershift-kubevirt-create-chain.yaml
being a single parametric step, see:
https://github.com/openshift/release/pull/50479

Then this could be easily added on top of that step.